flepimop/gempyor_pkg/tests/parameters/test_parameters_class.py

```diff
@@ -237,10 +237,9 @@ def test_timeseries_parameter_has_insufficient_columns_value_error(
         with pytest.raises(
             ValueError,
             match=(
-                rf"^ERROR loading file {tmp_file} for parameter sigma\: the number of "
-                rf"non 'date'\s+columns are {actual_columns}, expected "
-                rf"{mock_inputs.number_of_subpops()} \(the number of subpops\) or "
-                rf"one\.$"
+                rf"^Issue loading file '{tmp_file}' for parameter 'sigma': " 
+                rf"the number of non-'date' columns is '{actual_columns}', expected '{mock_inputs.number_of_subpops()}' " 
+                rf"\(number of subpopulations\) or one\.$"
```

Comment:
This is a very prescriptive error message match - which means any future tweaks to that error message have to be also maintained with changes here.

Is there something that we want to continue to ensure appears in the error message (e.g. something to do with variable introspection) without specifying the *exact* error message?